### PR TITLE
Create view for registrations monitoring

### LINKED ISSUE #2067: Create view for registrations monitoring in aggregator
## Why
We want to provide a simple query to retrieve the registrations aggregated data which is currently done with a complex SQL query as detailed in the [Registrations monitoring](https://github.com/input-output-hk/mithril/tree/main/docs/runbook/registrations-monitoring) runbook 

## What
Create a view replacing the complex SQL query for aggregated registrations

## How

- [x] Create a migration for the creation of the view
- [x] Update the [Registrations monitoring](https://github.com/input-output-hk/mithril/tree/main/docs/runbook/registrations-monitoring) runbook

### PR DESCRIPTION
**Pull request by sfauvel**
## Content

Create a view replacing the complex SQL query for aggregated registrations

This PR includes the view creation and the runbook update

## Pre-submit checklist

- Branch
  - [x] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [ ] CHANGELOG file is updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [ ] Update README file (if relevant)
  - [ ] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

## Issue(s)

Closes #2067

### CODE REVIEW

File: mithril-aggregator/src/event_store/database/repository.rs [hunk 1]
```diff
@@ -146,4 +146,183 @@ mod tests {
...
+            let _ = signer_routes::handlers::send_registration_event(
+                &transmitter_service,
+                &signer,
+                Some(signer_node_version.to_string()),
+                epoch,
+            );
+
+            let message: EventMessage = rx.try_recv().unwrap();
+
+            let _event = persister.persist(message).unwrap();
```
Comment (jpraynaud) [inline mithril-aggregator/src/event_store/database/repository.rs:191]: Isn't it easier to just persist the event message without tx/rx with the channel?

File: mithril-aggregator/src/event_store/database/migration.rs [hunk 2]
```diff
@@ -33,5 +33,53 @@ group by action, date;
...
+            3,
+            r#"
+create view stake_signer_version as with
+  signer_version as (
+    select
+       content->>'$.content.party_id' as party_id,
+       content->>'$.headers.signer-node-version' as node_version,
+       content->>'$.headers.epoch' as epoch,
+       content->>'$.content.stake' as stakes
+    from event
```
Comment (sfauvel) [inline mithril-aggregator/src/event_store/database/migration.rs:46]: We may add a `where` on `source` and/or `action`.
What do you think

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Review by dlachaume — APPROVED** [review]
LGTM 👍

**Review by Alenar — APPROVED** [review]

**Pull request merged**